plugins/pci-psp/fu-plugin-pci-psp.c

```diff
@@ -226,6 +231,7 @@ fu_plugin_add_security_attrs_rpmc(const gchar *path, FuSecurityAttrs *attrs)
 	if (!val) {
 		g_debug("no RPMC compatible SPI rom present");
 		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_ENABLED);
+		fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_OEM_FW_UPDATE);
```

Comment:
I don't think this is something the OEM can fix.  They picked an RPMC part or they didn't.

Reply:
Ack, thanks.